src/components/AppsPage/index.jsx

```diff
@@ -366,7 +366,7 @@ class AppsPage extends React.Component {
                 <h4>Database Support</h4>
                 <Tooltip
                   showIcon
-                  message="TODO: edit this explanation / info"
+                  message="Would you like to enable storage capability for your deployment?"
```

Comment:
how about `does your application need a database?`

Reply:
Yeah. Sounds simple and straight forward. Thanks.

Reply:
@mrwagaba .. this has been changed.
![Peek 2020-05-08 12-16](https://user-images.githubusercontent.com/29985169/81391509-e03af880-9125-11ea-98a7-7b4b0be21416.gif)